### ISSUE TITLE
[Accessibility- Script C 7.1] News_application- Some buttons don't contain a title

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3059]
**Environment:** Tribe

**Current behavior:**

Some buttons in the News application have no labels:

- The button that opens the filter of articles by spaces

- The button that opens the filter of articles

**Expected behavior:**

- Add aria-label or title attribute to buttons without label 
- 
**Debate topic**

Which attribute must be used as a title for buttons (title or aria-label)?

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
